src/adapter/src/sink_connection.rs

```diff
@@ -317,14 +288,13 @@ async fn build_kafka(
     Ok(StorageSinkConnection::Kafka(KafkaSinkConnection {
         connection: builder.connection,
         options: builder.options,
-        topic,
-        topic_prefix: builder.topic_prefix,
+        topic: builder.topic_name.clone(),
         relation_key_indices: builder.relation_key_indices,
         key_desc_and_indices: builder.key_desc_and_indices,
         value_desc: builder.value_desc,
         published_schema_info,
         consistency,
-        exactly_once: builder.reuse_topic,
+        exactly_once: true,
```

Comment:
Can this parameter be removed?

Reply:
We _could_, but IIUC there's some chance we'll re-introduce a not-exactly-once option in the future, so we're not quite as keen to rip that stuff out. (Whereas we're pretty confident that we won't ever do the topic-nonce-suffix thing again.)